README.md

```diff
@@ -45,10 +45,12 @@ The following set of features makes Artipie unique among all others:
 
 # Quickstart
 
-The fastest way to start using Artipie is via
-[Docker-compose](https://docs.docker.com/compose/), use 
-[`docker-compose.yaml`](https://github.com/artipie/artipie/blob/master/docker-compose.yaml) to run 
-the service:
+Make sure you have already installed both [Docker Engine](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/).
+Then, obtain [`docker-compose.yaml`](https://github.com/artipie/artipie/blob/master/docker-compose.yaml) file
```

Comment:
[`docker-compose.yaml`] -> [docker-compose.yaml] (without `)
Otherwise, the link doesn't look like other links.
Or delete the link because it already exists in the second line.